src/cli_parser.cpp

```diff
@@ -42,7 +42,7 @@ void CLIParser::parse(int argc, char** argv) {
                 "Show binary version")
             ("port",
                 "gRPC server port",
-                cxxopts::value<uint32_t>()->default_value("9178"),
+                cxxopts::value<uint32_t>()->default_value("0"),
```

Comment:
update all the documentation? --help will change

Reply:
Updated ovms.h

Reply:
Is it the only documentation we have about starting grpc server?
If so, maybe we should document the new feature?